server/resp/src/test/java/org/infinispan/server/resp/JsonCommandsTest.java

```diff
@@ -357,14 +354,12 @@ public void testJSONSETUndefinitePathError() {
       JsonValue jv1 = defaultJsonParser.createJsonValue("1");
       assertThatThrownBy(() -> {
          redis.jsonSet(key, jp1, jv1);
-      }).isInstanceOf(RedisCommandExecutionException.class)
-            .hasMessageStartingWith("ERR ");
+      }).isInstanceOf(RedisCommandExecutionException.class).hasMessageStartingWith("ERR ");
       JsonPath jp2 = new JsonPath("$..[0]");
       JsonValue jv2 = defaultJsonParser.createJsonValue("2");
       assertThatThrownBy(() -> {
          redis.jsonSet(key, jp2, jv2);
-      }).isInstanceOf(RedisCommandExecutionException.class)
-            .hasMessageStartingWith("ERR ");
+      }).isInstanceOf(RedisCommandExecutionException.class).hasMessageStartingWith("ERR ");
```

Comment:
IMO we need to check the errors with more than ERR because sometimes might fail for a reason that is not the one we want